internal/contour/virtualhost.go

```diff
@@ -189,10 +212,9 @@ func action(i *v1beta1.Ingress, be *v1beta1.IngressBackend) *route.Route_Route {
 
 // actioncrd computes the cluster route action, a *v2.Route_route for the
 // supplied ingress and backend
-func actioncrd(namespace string, be []v1alpha1.Upstream) *route.Route_Route {
+func actioncrd(namespace string, be []v1alpha1.Upstream) (*route.Route_Route, *k8s.RouteStatus) {
```

Comment:
Looks like we are always returning nil for `*k8s.RouteStatus`?

Reply:
yes we are, there aren't any "error" yet implemented for valid checking